bin/runtime/src/lib.rs

```diff
@@ -417,6 +432,8 @@ parameter_types! {
     pub const MaxPointsToBalance: u8 = 10;
 }
 
+#[cfg(feature = "liminal")]
+use pallet_feature_control::Feature;
```

Comment:
could we move those uses up? :D

Reply:
oh, I believed that my built-in formatter will do it automatically... anyway moved up